packages/lib/server-only/admin/get-users-stats.ts

```diff
@@ -16,3 +18,73 @@ export const getUsersWithSubscriptionsCount = async () => {
     },
   });
 };
+
+export const getUserWithAtLeastOneDocumentPerMonth = async () => {
+  return await prisma.user.count({
+    where: {
+      Document: {
+        some: {
+          createdAt: {
+            gte: new Date(new Date().setMonth(new Date().getMonth() - 1)),
+          },
+        },
+      },
+    },
+  });
+};
+
+export const getUserWithAtLeastOneDocumentSignedPerMonth = async () => {
+  return await prisma.user.count({
+    where: {
+      Document: {
+        some: {
+          status: {
+            equals: DocumentStatus.COMPLETED,
+          },
+          createdAt: {
+            gte: new Date(new Date().setMonth(new Date().getMonth() - 1)),
+          },
+        },
+      },
+    },
+  });
+};
```

Comment:
This feels wrong, or at least the naming does

Are you looking for users who are recipients and signed a document?

Or a user who was simply part of a document that was completed?

Reply:
I'm looking for users who have uploaded a document in the past month and the document has been signed. maybe the naming feels wrong

Reply:
"Users that created at least 1 document and had it completed" maybe?

Reply:
@ElTimuro the current query is for the completed documents in the past month, should it stay the same? or make the query for all the document?